bluebrain/repo-bluebrain/packages/py-multiscale-run/package.py

```diff
@@ -15,6 +15,8 @@ class PyMultiscaleRun(PythonPackage):
     maintainers("tristan0x", "cattabiani")
 
     version("develop", branch="main")
+    version("0.5.1", tag="0.5.1")
+    version("0.5", tag="0.5")
```

Comment:
I'll allow it. But for future reference, we don't need intermediate versions :)

Reply:
tristan is annoyed that I created a 0.5.1 tag without a 0.5. The 0.5.1 is here. Anyway I move from here somebody complains

Reply:
Yes, it should also be `0.5.0` not `0.5` :)

I'm done now :)

Reply:
what if I just remove it? It has some problems anyway that version

Reply:
Yes, remove faulty versions and towers.